examples/rvc-app/linux/README.md

```diff
@@ -0,0 +1,140 @@
+# Matter Linux Robotic Vacuum Cleaner Example
+
+An example showing the use of Matter on Linux. The document will describe how to
+build and run Matter Linux Robotic Vacuum Cleaner Example on Raspberry Pi. This
+doc is tested on **Ubuntu for Raspberry Pi Server 20.04 LTS (aarch64)** and
+**Ubuntu for Raspberry Pi Desktop 20.10 (aarch64)**
+
+To cross-compile this example on x64 host and run on **NXP i.MX 8M Mini**
+**EVK**, see the associated
+[README document](../../../docs/guides/nxp/nxp_imx8m_linux_examples.md) for
+details.
+
+<hr>
+
+-   [Matter Linux Robotic Vacuum Cleaner Example](#matter-linux-robotic-vacuum-cleaner-example)
+    -   [Building](#building)
+    -   [Commandline Arguments](#commandline-arguments)
+    -   [Running the Complete Example on Raspberry Pi 4](#running-the-complete-example-on-raspberry-pi-4)
+    -   [Running RPC console](#running-rpc-console)
+    -   [Device Tracing](#device-tracing)
+
+<hr>
+
+## Building
+
+-   Install tool chain
+
+          $ sudo apt-get install git gcc g++ python pkg-config libssl-dev libdbus-1-dev libglib2.0-dev ninja-build python3-venv python3-dev unzip
+
+-   Build the example application:
+
+          $ cd ~/connectedhomeip/examples/rvc-app/linux
+          $ git submodule update --init
+          $ gn gen out/debug
+          $ ninja -C out/debug
+
+-   To delete generated executable, libraries and object files use:
+
+          $ cd ~/connectedhomeip/examples/rvc-app/linux
+          $ rm -rf out/
+
+-   Build the example with pigweed RPC
+
+          $ cd ~/connectedhomeip/examples/rvc-app/linux
+          $ git submodule update --init
+          $ source third_party/connectedhomeip/scripts/activate.sh
+          $ gn gen out/debug --args='import("//with_pw_rpc.gni")'
+          $ ninja -C out/debug
+
+## Commandline arguments
+
+-   `--wifi`
+
+    Enables WiFi management feature. Required for WiFi commissioning.
+
+-   `--thread`
+
+    Enables Thread management feature, requires ot-br-posix dbus daemon running.
+    Required for Thread commissioning.
+
+-   `--ble-device <interface id>`
+
+    Use specific bluetooth interface for BLE advertisement and connections.
+
+    `interface id`: the number after `hci` when listing BLE interfaces by
+    `hciconfig` command, for example, `--ble-device 1` means using `hci1`
+    interface. Default: `0`.
+
+## Running the Complete Example on Raspberry Pi 4
+
+> If you want to test Echo protocol, please enable Echo handler
```

Comment:
why is the echo protocol referenced here?

I see this in a lot of README.md inside linux ... but generally it seems out of place.

Reply:
This is a copy/paste from another example. I've removed this section.